HpToolsLauncher/TestSuiteRunResults.cs

```diff
@@ -53,6 +54,12 @@ public int NumTests
             set { m_numTests = value; }
         }
 
+        public int NumWarnings
+		{
+            get { return m_numWarnings; }
+            set { m_numWarnings = value; }
```

Comment:
Number of generated warnings was not implemented on the TestSuiteRunResults side.